# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/consolekit/hotfix-autogen.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- ConsoleKit2-1.2.6/autogen.sh.vanilla	2025-10-12 20:47:26.287091788 +0200
+++ ConsoleKit2-1.2.6/autogen.sh	2025-10-12 20:47:46.468865374 +0200
@@ -34,10 +34,6 @@
 conf_flags="--enable-maintainer-mode --cache-file=config.cache --disable-static"
 
 if test x$NOCONFIGURE = x; then
-  echo Running $srcdir/configure $conf_flags "$@" ...
-  $srcdir/configure $conf_flags "$@" \
-  && echo Now type \`make\' to compile. || exit 1
-else
   echo Skipping configure process.
 fi
 
